src/main/groovy/de/dkfz/roddy/tools/versions/Version.groovy

```diff
@@ -151,6 +151,20 @@ class Version implements Comparable<Version> {
         return "${major}.${minor}.${patch}-${revision}"
     }
 
+    String toString(VersionLevel level) {
+        def builder = new StringBuilder()
```

Comment:
Could this not be a simple for loop?

for(l in VersionLevel.MAJOR..level) {
if else ... 
}

just seems one step too much :D

Reply:
fixed